testing/unittests/jstreamtests.cpp

```diff
@@ -1029,28 +1031,32 @@ class JlibStreamStressTest : public CppUnit::TestFixture
 
         {
             CCycleTimer timer;
-            Owned<IBufferedSerialInputStream> in = createBufferedSerialInputStream(buffer.reset(0));
+            MemoryBuffer clone(buffer.length(), buffer.toByteArray());
```

Comment:
Since the introduction of the option to fill memory after a read these unit tests were failing.